### ISSUE TITLE
Real-World Exp hyperparameters

### ISSUE THREAD

**Issue by armanakbari** [Robbyant/lingbot-va#95]
Thanks for the great work. I have fine-tuned on some real-world tasks. How do you find the best parameters for attn_window, chunk size, number of latents in each chunk, number of action in each latent (16)?

In my experiments pi 0.5 is still performing better. I have fine-tune on each task for 3k to 5k seperately and the percision is not there! Success rate is somewhere around 30% or 40% and for tasks that require more precision it drops

**Comment by aopolin-lv** [comment]
> Thanks for the great work. I have fine-tuned on some real-world tasks. How do you find the best parameters for attn_window, chunk size, number of latents in each chunk, number of action in each latent (16)?
> 
> In my experiments pi 0.5 is still performing better. I have fine-tune on each task for 3k to 5k seperately and the percision is not there! Success rate is somewhere around 30% or 40% and for tasks that require more precision it drops

same issue, my config is as follow:
| Parameter | Value |
| :--- | :---: |
| batch size | 64 |
| learning rate | 1e-5 |
| training steps | 5k |
| frame chunk size | 4 |
| action per frame |12 |
| num_eps | 480 |

<img width="1179" height="314" alt="Image" src="https://github.com/user-attachments/assets/d2d1dabc-8eba-4d27-88b8-4c6bfe6790f7" />
